lib/viewing/ViewRelationship.js

```diff
@@ -141,6 +175,18 @@ ViewRelationship.prototype = {
 
 };
 
+function chooseDestination(destinationEl, destination) {
+    if (_.isString(destination)) {
```

Comment:
maybe drop underscore and use `typeof destination === "string"`?

Reply:
I went with underscore because underscore is forced with backbone - since I already have it, might as well use it